jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It is fine to add a VS.code specific file, but maybe do it in a different PR?
Also, if there are other config options needed, feel free to add them to the file.